lib/tools/system-calls.js

```diff
@@ -411,6 +412,20 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   return proc;
 };
 
+systemCallMethods.getAdbVersion = async function () {
```

Comment:
I was just about to ask how it was envisioned this would be used. If we need to access it piecemeal, I agree this would be better, so the parsing doesn't have to happen all over the place. I would add `versionFloat: 1.1` and `versionString: "1.1.2"`.